### ISSUE TITLE
[BUG] Telegram Bot "is not a member of the channel" (but, it is)

### ISSUE THREAD

**Issue by Plop** [kvangent/PokeAlarm#222]
## Problem Description

I'm trying to use this with a Telegram Bot, on a channel, but I can't make it work, even if 
1) the channel is created
2) the config seems right
3) ???
## Your Environment

Operating System:  Debian 8
Python Version: (use `python -V`)  python 2.7.9
PokemonGo-Map startup command: `<cmd>`  
PokeAlarm startup command: `<cmd>` python runwebhook.py -L fr

Did the issue persist after restarting the script?: yes

## Console Output

```
PokeAlarm# python runwebhook.py -L fr
2016-09-13 09:28:15,236 [     alarm_manager] [   INFO] You will be notified of the following pokemon:
[...]
2016-09-13 09:28:15,238 [     alarm_manager] [   INFO] Mew (Unl)
2016-09-13 09:28:15,238 [     alarm_manager] [   INFO] Alarm not activated: boxcar because value not set to "True"
2016-09-13 09:28:16,561 [  pushbullet_alarm] [   INFO] Pushing to channel None.
2016-09-13 09:28:16,561 [  pushbullet_alarm] [   INFO] Pushing to channel None.
2016-09-13 09:28:16,562 [  pushbullet_alarm] [   INFO] Pushing to channel None.
2016-09-13 09:28:17,546 [  pushbullet_alarm] [   INFO] Pushbullet Alarm intialized.
2016-09-13 09:28:17,546 [     alarm_manager] [   INFO] Alarm not activated: pushover because value not set to "True"
2016-09-13 09:28:17,546 [     alarm_manager] [   INFO] Alarm not activated: slack because value not set to "True"
2016-09-13 09:28:17,557 [    connectionpool] [   INFO] Starting new HTTPS connection (1): api.telegram.org
Traceback (most recent call last):
  File "runwebhook.py", line 93, in <module>
    alarm_thread = Alarm_Manager()
  File "/var/www/PokeAlarm/alarms/alarm_manager.py", line 61, in __init__
    self.alarms.append(Telegram_Alarm(alarm))
  File "/var/www/PokeAlarm/alarms/Telegram/telegram_alarm.py", line 53, in __init__
    self.client.sendMessage(self.pokemon['chat_id'], 'PokeAlarm activated! We will alert this chat about pokemon.')
  File "/usr/local/lib/python2.7/dist-packages/telepot/__init__.py", line 240, in sendMessage
    return self._api_request('sendMessage', _rectify(p))
  File "/usr/local/lib/python2.7/dist-packages/telepot/__init__.py", line 229, in _api_request
    return api.request((self._token, method, params, files), **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/telepot/api.py", line 131, in request
    return _parse(r)
  File "/usr/local/lib/python2.7/dist-packages/telepot/api.py", line 126, in _parse
    raise exception.TelegramError(description, error_code, data)
telepot.exception.TelegramError: (u'Forbidden: bot is not a member of the channel chat', 403, {u'error_code': 403, u'ok': False, u'description': u'Forbidden: bot is not a member of the channel chat'})

```

## Your Alarms Config

```json
    {
                        "active": "True",
                        "type":"telegram",
                        "chat_id":"@CHANNEL_NAME",
                        "bot_token":"BOT_TOKEN",
                                "header":"Un <pkmn> sauvage est apparu !",
                                "body_text":"<gmaps> \n Disponible jusqu'à <24h_time> (<time_left>)."
        },

```

## Troubleshooting Checklist
[x] Most recent version of PokemonGo-Map
[x] Most recent version of PokeAlarm
[x] Checked the Wiki page for service in question

**Comment by Spiritvn** [comment]
Did you add the bot to be channel admin yet?

**Comment by Plop** [comment]
Hi, 

Yes, the bot is channel admin, he also is in the member list... That's Why I don't understand why I got this error

**Comment by Plop** [comment]
![2016-09-13_14 08 22](https://cloud.githubusercontent.com/assets/876470/18473170/d7f161a8-79bb-11e6-9449-5db4c951a4c5.jpg)


There it is

**Comment by reme-dy** [comment]
```
    {
                        "active": "True",
                        "type":"telegram",
                        "chat_id":"BOT_CHAT_ID_HERE",
                        "bot_token":"BOT_TOKEN",
                                "header":"Un <pkmn> sauvage est apparu !",
                                "body_text":"<gmaps> \n Disponible jusqu'à <24h_time> (<time_left>)."
        },
```

At the alert level (same as telegram), `"chat_id"` is the bot's identifier, which you got when you checked `https://api.telegram.org/bot<BOT_TOKEN_HERE>/getUpdates`.  It's sort of like the bot's chat room name. 

At the `pokemon`, `pokestop` or `gym` level in the alarm code, `chat_id` is your `@channelname` of the public channel you created and added your bot to.

Follow step 4 in [How to get a Telegram API Key](https://github.com/kvangent/PokeAlarm/wiki/Telegram#how-to-get-a-telegram-api-key) if you haven't got the bot's chat_id.

**Comment by Plop** [comment]
Well, I've been struggling a little, but I finally succeeded.
I've been following (again) the tutorial, to set the chat_it back to mine.
On Telegram, I set the channel in public, changed its name, and then put it on private again.
I've then send a message on the channel using this :
`curl -X POST "https://api.telegram.org/botXXX:YYYY/sendMessage" -d "chat_id=@CHANNEL_NAME&text=my sample text"
`
I got the chat_id from the channel (like this : -XXXXXXXX), and used it in the "pokemon"{} bloc, as the chat_id.
Everything working fine now :) Thanks !
